### PR TITLE
Added Backstage version to info command

### PR DESCRIPTION
**Pull request by awanlin**
## Hey, I just made a Pull Request!

Added Backstage version to `yarn backstage-cli info` command output:

image

If there is no `backstage.json` this will fallback to "N/A"

#### :heavy_check_mark: Checklist

- [x] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [x] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: packages/cli/src/commands/info.ts [hunk 1]
```diff
@@ -19,17 +19,30 @@ import os from 'os';
...
   await new Promise(async () => {
     const yarnVersion = await runPlain('yarn --version');
     // eslint-disable-next-line no-restricted-syntax
-    const isLocal = require('fs').existsSync(paths.resolveOwn('./src'));
+    const isLocal = fs.existsSync(paths.resolveOwn('./src'));
+
+    const backstageFile = paths.resolveTargetRoot('backstage.json');
+    let backstageJson = undefined;
+    if (fs.existsSync(backstageFile)) {
+      const buffer = await fs.readFile(backstageFile);
```
Comment (benjdlambert) [inline packages/cli/src/commands/info.ts:33]: Think you can use `fs.readJSON` here instead. And wrap it in a `try/catch` just incase it's invalid 👍

**Review by benjdlambert — CHANGES_REQUESTED** [review]
Thanks for this! Nice idea! Just a quick comment about the reading :)

File: packages/cli/src/commands/info.ts (hunk 1 above)
Comment (awanlin) [reply packages/cli/src/commands/info.ts]: Sure, I've made this change, how does that looks to you?

File: packages/cli/src/commands/info.ts [hunk 2]
```diff
@@ -19,17 +19,30 @@ import os from 'os';
 import { runPlain } from '../lib/run';
 import { paths } from '../lib/paths';
 import { Lockfile } from '../lib/versioning';
+import fs from 'fs-extra';
 
 export default async () => {
   await new Promise(async () => {
     const yarnVersion = await runPlain('yarn --version');
     // eslint-disable-next-line no-restricted-syntax
```
Comment (benjdlambert) [inline packages/cli/src/commands/info.ts:27]: Think maybe you can also remove this line too? Not sure why this is valid anymore.

File: packages/cli/src/commands/info.ts [hunk 3]
```diff
@@ -19,17 +19,30 @@ import os from 'os';
...
+    const isLocal = fs.existsSync(paths.resolveOwn('./src'));
+
+    const backstageFile = paths.resolveTargetRoot('backstage.json');
+    let backstageVersion = 'N/A';
+    if (fs.existsSync(backstageFile)) {
+      try {
+        const backstageJson = await fs.readJSON(backstageFile);
+        backstageVersion = backstageJson.version ?? 'N/A';
+      } catch (error) {
+        backstageVersion = 'N/A';
```
Comment (benjdlambert) [inline packages/cli/src/commands/info.ts:37]: Maybe log as warning if you get here?

**Review by benjdlambert — COMMENTED** [review]
Looking good @awanlin! Just a few more comments :pray:

File: packages/cli/src/commands/info.ts (hunk 2 above)
Comment (awanlin) [reply packages/cli/src/commands/info.ts]: For sure, pulled it out

File: packages/cli/src/commands/info.ts (hunk 3 above)
Comment (benjdlambert) [reply packages/cli/src/commands/info.ts]: Nice! Thanks!

**Review by benjdlambert — APPROVED** [review]

**Comment by benjdlambert** [comment]
Thank you @awanlin!

**Comment by awanlin** [comment]
Thanks @benjdlambert (also for that typo fix 😉 )

**Comment by benjdlambert** [comment]
Ahah! Nice OK! Perfect, thats good to know at least. :pray:

**Comment by benjdlambert** [comment]
No worries :pray:

**Pull request merged**